lib/classes/TradeOffer.js

```diff
@@ -263,7 +262,7 @@ function addItem(details, offer, list) {
 		assetid: (details.assetid || details.id).toString(), // always needs to be a string
 		appid: parseInt(details.appid, 10), // always needs to be an int
 		contextid: details.contextid.toString(), // always needs to be a string
-		amount: parseInt(details.amount || 1, 10), // always needs to be an int
+		amount: parseInt(details.amount || 1, 10) // always needs to be an int
```

Comment:
We could replace `parseInt` with `string << 0`, which is 2.5x faster than parse int? It seems to work with all cases.
```javascript
3 << 0 // 3
3.33 << 0 // 3
"3" << 0 // 3
"3.33" << 0 // 3
```
[source](http://phrogz.net/js/string_to_number.html)